### DISCUSSION TITLE
Azure Blob for Media/Static Storage

### DISCUSSION THREAD

**Discussion by inf3cti0n95** [saleor/saleor#14430] [Ideas]
### What I'm trying to achieve
We're using Azure for Cloud services, and wanted to used Azure Blob, which is MSFT alternative to S3 for cloud storage of Media and Static Files.

### Describe a proposed solution
I think, since the Azure Blob works same way as s3 it should not be an issue, configuring it.

**Comment by peelar** [comment]
Hello @inf3cti0n95 👋 

Thanks for your request. I don't think we plan to provide such integration in the nearest future, but we noted it and will keep it in mind.
